installing/overview/cluster-capabilities.adoc

```diff
@@ -32,7 +32,7 @@ include::modules/cluster-bare-metal-operator.adoc[leveloffset=+2]
 
 [role="_additional-resources"]
 .Additional resources
-* xref:../../installing/installing_bare_metal_ipi/ipi-install-overview.adoc#ipi-install-overview[Deploying installer-provisioned clusters on bare metal]
+* xref:../../installing/installing_bare_metal_ipi/ipi-install-overview.adoc#ipi-install-overview[Installing on bare metal with the IPI Installer]
```

Comment:
The original phrasing is more accurate

Reply:
Original phrasing is, however, inconsistent with all other titles that begin with `Installing...`

Reply:
Addressed